src/main/scala/mesosphere/marathon/api/akkahttp/EntityMarshallers.scala

```diff
@@ -136,4 +184,8 @@ object EntityMarshallers {
     case ValidationFailed(failure) =>
       complete(StatusCodes.UnprocessableEntity -> failure)
   }
+
+  import scala.language.implicitConversions
+  implicit def entityMarshallerToMessageUnmarshaller[T](um: FromEntityUnmarshaller[T]): FromMessageUnmarshaller[T] =
```

Comment:
seems like the name here should be `entityUnmarshallerToMessageUnmarshaller`

Reply:
also, how bad would it be if this conversion were explicit? My default is to avoid implicit conversions.

Reply:
+1 on that. Let's try to avoid the implicit conversion unless we can find a great benefit.